Xamarin.Forms.Core/Grid.cs

```diff
@@ -212,6 +212,11 @@ internal override void ComputeConstraintForView(View view)
 			view.ComputedConstraint = result;
 		}
 
+		[EditorBrowsable(EditorBrowsableState.Never)]
+		public void InvalidateMeasureInernalNonVirtual(InvalidationTrigger trigger)
```

Comment:
As far as I can tell, the only reason we explicitly implemented the methods on the controller interface was to hide them from isense. Unfortunately, this also had the side effect of breaking all our call sites and requiring us to sprinkle the idiom Foo.Bar -> ((IController)Foo).Bar everywhere. By using the browser attribute we get the desired design effect (hide from isense) but we don't break our call sites which can simply remain Foo.Bar. So I'd actually suggest we revert all explicit implementations and add the browser attribute and at the same time revert the callsites back from ((IController)Foo).Bar to simply Foo.Bar.